stable/mediawiki/templates/NOTES.txt

```diff
@@ -1,15 +1,46 @@
 {{- if or .Values.mariadb.enabled .Values.externalDatabase.host -}}
 
-** Please be patient while the chart is being deployed **
+{{- if empty (include "mediawiki.host" .) -}}
+###############################################################################
+### ERROR: You did not provide an external host in your 'helm install' call ###
+###############################################################################
 
-{{- if .Values.ingress.enabled }}
```

Comment:
This conditional `{{- if .Values.ingress.enabled }}` disappeared from the NOTES.txt.
It should be part of the `{{- else }}` condition once the Medawiki hostname is indicated.